### ISSUE TITLE
Make Installation of Windows Terminal and PowerShell Optional

### ISSUE THREAD

**Issue by RoyiAvital** [kpcintegrations/KPC-OneCommanderScripts#1]
The current installation script installs Windows Terminal and the latest PowerShell with no notification or request from user.

I think it is better to ask fro user permission to do so.

**Comment by kamekairos** [comment]
I agree, I will be adding a prompt dialog, although without some version of windows terminal installed and at least PowerShell 7 the script pack will not work. I was trying to make it easier on less tech savvy people. I will update and close this issue when done. Thanks for the feedback.

**Comment by kamekairos** [comment]
Added confirmation dialogs to install both programs however the script pack is only verified working on Powershell 7.4+ being installed by this script or already installed and on path.
